src/main/java/in/projecteka/gateway/SecurityConfiguration.java

```diff
@@ -26,22 +26,7 @@
 import java.text.ParseException;
 import java.util.ArrayList;
 
-import static in.projecteka.gateway.common.Constants.V_1_CARE_CONTEXTS_DISCOVER;
-import static in.projecteka.gateway.common.Constants.V_1_CARE_CONTEXTS_ON_DISCOVER;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENTS_FETCH;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENTS_HIP_NOTIFY;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENTS_HIU_NOTIFY;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENTS_ON_FETCH;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENT_REQUESTS_INIT;
-import static in.projecteka.gateway.common.Constants.V_1_CONSENT_REQUESTS_ON_INIT;
-import static in.projecteka.gateway.common.Constants.V_1_LINKS_LINK_CONFIRM;
-import static in.projecteka.gateway.common.Constants.V_1_LINKS_LINK_INIT;
-import static in.projecteka.gateway.common.Constants.V_1_LINKS_LINK_ON_CONFIRM;
-import static in.projecteka.gateway.common.Constants.V_1_LINKS_LINK_ON_INIT;
-import static in.projecteka.gateway.common.Constants.V_1_PATIENTS_FIND;
-import static in.projecteka.gateway.common.Constants.V_1_PATIENTS_ON_FIND;
-import static in.projecteka.gateway.common.Constants.V_1_HEALTH_INFORMATION_CM_REQUEST;
-import static in.projecteka.gateway.common.Constants.V_1_HEALTH_INFORMATION_HIP_REQUEST;
+import static in.projecteka.gateway.common.Constants.*;
```

Comment:
It's good to import only required constants instead of whole file.

Reply:
done